save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/ProjectSettingsMenu.kt

```diff
@@ -172,6 +174,19 @@ fun projectSettingsMenu(
                         }
                     }
                 }
+                div("row d-flex align-items-center mt-2 mr-2 ml-2") {
+                    div("col-5 text-left") {
+                        +"User settings:"
+                    }
+                    div("col-7 row") {
+                        button(type = ButtonType.button, classes = "btn btn-sm btn-primary") {
+                            attrs.onClickFunction = {
+                                updateGit(true)
+                            }
+                            +"Add"
```

Comment:
Please rename it properly “Add” -> “Add github credentials”

Reply:
Renamed